plugins/net_plugin/net_plugin.cpp

```diff
@@ -420,8 +420,8 @@ namespace eosio {
 
 
       struct chain_info_t {
-         block_id_type lib_id;
-         uint32_t      lib_num = 0;
+         block_id_type fork_root_id;
+         uint32_t      fork_root_num = 0;
```

Comment:
why not `froot`?

Reply:
I was matching the existing `fork_head_` members.